packages/editor/src/components/warning/index.js

```diff
@@ -7,23 +7,53 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { Children } from '@wordpress/element';
+import { Dropdown, IconButton, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
 
-function Warning( { className, actions, children } ) {
+function Warning( { className, primaryActions, children, hiddenActions } ) {
 	return (
 		<div className={ classnames( className, 'editor-warning' ) }>
 			<div className="editor-warning__contents">
 				<p className="editor-warning__message">{ children }</p>
 
-				{ Children.count( actions ) > 0 && (
+				{ Children.count( primaryActions ) > 0 && (
 					<div className="editor-warning__actions">
-						{ Children.map( actions, ( action, i ) => (
+						{ Children.map( primaryActions, ( action, i ) => (
 							<span key={ i } className="editor-warning__action">
 								{ action }
 							</span>
 						) ) }
 					</div>
 				) }
 			</div>
+
+			{ hiddenActions && (
+				<div className="editor-warning__hidden">
+					<Dropdown
+						className="edit-post-more-menu"
```

Comment:
The `edit-post-*` className seems useless right? (they don't follow our guidelines neither)

Reply:
It's the same class used in https://github.com/WordPress/gutenberg/blob/master/edit-post/components/header/more-menu/index.js, making the ellipsis vertical rather than horizontal. I can file a separate issue if that also needs changing to match the guidelines?

Reply:
I don't think we should rely on classnames styled in other components. We should aim to reuse components instead of classnames instead.

So two options:

 - I'm fine duplicating the styles (if not too much) and using the right classnames here `editor-warning__*`
 - Creating a generic component with these styles but in that case, this component should be meaningful. Maybe it could be a prop of `Dropdown` instead?